sdk/daml-lf/snapshot/src/benchmark/scala/com/digitalasset/daml/lf/testing/snapshot/ReplayBenchmark.scala

```diff
@@ -60,3 +68,15 @@ class ReplayBenchmark {
   }
 
 }
+
+object ReplayBenchmark {
+  @State(Scope.Thread)
+  @AuxCounters(AuxCounters.Type.EVENTS)
+  class EventCounter {
+    var stepCount: Long = 0
```

Comment:
Plan is to generally add other secondary metric data that we collect to this counter class